app/controllers/api/moves_controller.rb

```diff
@@ -38,19 +35,43 @@ def filtered
 
   private
 
+    def find_moves(active_record_relationships:)
+      Moves::Finder.new(
+        filter_params: filter_params,
+        ability: current_ability,
+        order_params: params[:sort] || {},
+        active_record_relationships: active_record_relationships,
+      ).call
+    end
+
     def moves
-      @moves ||= Moves::Finder.new(filter_params: filter_params,
-                                   ability: current_ability,
-                                   order_params: params[:sort] || {},
-                                   active_record_relationships: active_record_relationships).call
+      @moves ||= find_moves(active_record_relationships: active_record_relationships)
     end
 
     def validate_filter_params
       Moves::ParamsValidator.new(filter_params, params[:sort] || {}).validate!(action_name.to_sym)
     end
 
     PERMITTED_FILTER_PARAMS = %i[
-      date_from date_to created_at_from created_at_to date_of_birth_from date_of_birth_to location_type status from_location_id to_location_id location_id supplier_id move_type cancellation_reason rejection_reason has_relationship_to_allocation ready_for_transit
+      date_from
+      date_to
+      created_at_from
+      created_at_to
+      date_of_birth_from
+      date_of_birth_to
+      location_type
+      status
+      from_location_id
+      to_location_id
+      location_id
+      supplier_id
+      move_type
+      cancellation_reason
+      rejection_reason
+      has_relationship_to_allocation
+      ready_for_transit
+      profile_id
+      person_id
```

Comment:
I know it's technically a separate piece of work, but how straightforward would it be to support `reference` as a filter too?

Reply:
Since it's just a plain field on the move it would be very easy to do. I don't mind adding it as another commit in this PR.

Reply:
Added in d98bd8e.

Reply:
This will unlock a really nice option for us to provide a simple reference search to start with on the frontend 👍 

And also just allow us to debug queries using the API endpoints if all we have is a reference. Before that we had to search directly on the DB...